tests/integration/run_test.go

```diff
@@ -23,9 +23,12 @@ func runTests(t *testing.T, when spec.G, it spec.S) {
 
 	when("rio run is called", func() {
 		it("should create a service with default specifications", func() {
+			runningPods := service.GetRunningPods()
 			assert.Equal(t, 1, service.GetAvailableReplicas(), "should have one available replica")
 			assert.Equal(t, 100, service.GetCurrentWeight())
 			assert.Equal(t, "nginx", service.GetImage())
+			assert.Contains(t, runningPods, service.App.Name)
+			assert.Contains(t, runningPods, "2/2")
```

Comment:
Was thinking there should be better way to test this, but looks like ready pods summary isn't available via json without calculating yourself.

Reply:
Technically, we can validate this by getting the pods at `.items[*].status.containerStatuses` at either ready or state. I feel like in this case the 2/2 is a bit more clear and concise, but am open to changing it.